wyrekey.js

```diff
@@ -0,0 +1,13 @@
+function makeSecret(length) {
```

Comment:
What's that? Where is it used?

Reply:
whoops. this snuck in the PR. i'll remove